pkg/model/agent.go

```diff
@@ -0,0 +1,17 @@
+package model
+
+type Agent struct {
+	AgentIdentifier        string `json:"agentIdentifier,omitempty" yaml:"agentIdentifier,omitempty"`
+	AgentInstanceUuid      string `json:"agentInstanceUuid,omitempty" yaml:"agentInstanceUuid,omitempty"`
```

Comment:
```suggestion
	AgentInstanceUUID      string `json:"agentInstanceUuid,omitempty" yaml:"agentInstanceUuid,omitempty"`
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms